python/pylibcugraph/pylibcugraph/louvain.pyx

```diff
@@ -51,6 +52,7 @@ from pylibcugraph.utils cimport (
 def louvain(ResourceHandle resource_handle,
             _GPUGraph graph,
             size_t max_level,
+            double threshold,
```

Comment:
Interestingly, I could support double here.  Probably should make these consistent, left it in this odd state for discussion.

Reply:
Changed these to float